vlm/web_app.py

```diff
@@ -25,9 +33,23 @@ async def status(request: web.Request) -> web.Response:
     return web.json_response({'success': True})
 
 
+async def match(request: web.Request) -> web.Response:
+    return web.json_response(get_variant_match(request.query))
+
+
 async def init_web_app():
+    spark_conf = {}
+    # memory limits adapted from https://github.com/hail-is/hail/blob/main/hail/python/hailtop/hailctl/dataproc/start.py#L321C17-L321C36
+    if MACHINE_MEM:
+        spark_conf['spark.driver.memory'] = f'{int((int(MACHINE_MEM) - 11) * JVM_MEMORY_FRACTION)}g'
```

Comment:
The code to build `spark_conf` here looks identical to how it's done in the `hail_search` web_app, can this be put in a helper function somewhere?

Reply:
So ideally yes. However, the docker builds for hail_search and the vlm each only include the single directory for their respective apps, and then the unit tests and releases are only triggered on change to those specific directories. Creating a shared directory that is included in thh build for both apps and then updating the github triggers to include the additional shared  directory is possible, but it seemed like more work tahn it was worth to avoid this relatively small amout of copy-paste. However, if you and/or @bpblanken feel like its worth doing than I can do so. However, in the interest of keeping scope sane I would recommend that this gets merged first and then I do the shared code move in a subsequent PR

Reply:
Considering the amount of work it would take to make this code shared, I think it's fine if it's duplicated.